testing/matrix/linter/rules/rules.go

```diff
@@ -166,8 +172,22 @@ func containerEnvVariablesDuplicates(object storage.StoreObject, containers []v1
 	return errors.EmptyRuleError
 }
 
-func containerImageTagLatest(object storage.StoreObject, containers []v1.Container) errors.LintRuleError {
+func shouldSkipModuleContainer(module string, container string) bool {
+	if module == "okmeter" && container == "okagent" {
```

Comment:
It would be best to add comments on why these images are ignored.

Reply:
Added comments